bin/build

```diff
@@ -1,27 +1,45 @@
 #!/usr/bin/env bash
 
+DEBUG="$1"
+export DOCKER_BUILDKIT=1
+
 set -e
 
+cd $(dirname $0)/..
+
 . bin/build_utils
 
 FULL_VERSION_TAG="$(full_version_tag)"
 
 echo "---"
 
-echo "Building secrets-provider-for-k8s Docker image"
+if [ $DEBUG == "debug" ]; then
```

Comment:
if i understand correctly, you are planning to run this with `./bin/build debug`. Please see how we handle flags in [`test_integration`](https://github.com/cyberark/secrets-provider-for-k8s/blob/master/bin/test_integration) and maintain that behaviour here as well.